torch/_dynamo/output_graph.py

```diff
@@ -465,6 +476,23 @@ def wrap_name(module_key):
         for i in itertools.count():
             if name not in self.nn_modules:
                 self.nn_modules[name] = target
+                assert self.nn_modules_sources is not None
+                self.nn_modules_sources[name] = source
+                if isinstance(target, torch.nn.Module):
+                    # annoying, but there are cases when we do not have parameters
+                    # see test_nn_moduledict_contains
+                    if hasattr(target, "_parameters"):
+                        for n, p in target.named_parameters():
```

Comment:
so i buy that _parameters may not exist, but does .named_parameters() do something other than iterate an empty set in that case?

Reply:
It raises!